tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -462,8 +463,8 @@ var _ = Describe("odo devfile push command tests", func() {
 			utils.ExecWithInvalidCommandGroup(context, cmpName, namespace)
 		})
 
-		It("should not restart the application if restart is false", func() {
-			utils.ExecWithRestartAttribute(context, cmpName, namespace)
+		It("should not restart the application if it is hot reload capable", func() {
```

Comment:
Can we also consider default case. What if we don't set hot reload to `true`. According to https://github.com/devfile/api/pull/96/files#diff-75282a0f7931c780b57f6bf165b6fc9fR274-R277 the description says it should not restart with hot reload enable. However with default case it should. Correct me if I am wrong

Reply:
@prietyc123 sure, will add more tests.

Reply:
Done.